### PR TITLE
Catelogue item link

### PR DESCRIPTION
**Pull request by orangesmartie92**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -77,6 +78,11 @@ class App extends Component {
 
             <Route exact path="/browse" render={() => <BrowseItems />} />
 
+
+            <Route exact path="/browse/:productID" render={(routeProps) => {
+              console.log(routeProps);
```
Comment (jangeroo) [inline src/App.js]: Probably don't need to log this here. Can we comment it out?

File: src/ItemDetails.js [hunk 2]
```diff
@@ -1,42 +1,38 @@
...
+            )
+
+    }
+
+    render() {
+        return (
+            <div className="catelogue-container" key={this.state.item.blurb}>
+                <Catelogue item={this.state.item} />
+            </div>
+
```
Comment (jangeroo) [inline src/ItemDetails.js]: Remove blank line

File: src/ItemDetails.js [hunk 3]
```diff
@@ -1,42 +1,38 @@
...
-                            <p>$ {item.price} </p>
-                            {/* <button className="btn btn-danger btn-sm" onClick={this.removeItem.bind(this, index)}><span className="glyphicon glyphicon-trash"></span> Delete</button> */}
-                            </li>
-                        )}
-                    </ul>
-                </div>
-            );
-        }
+            )
+
```
Comment (jangeroo) [inline src/ItemDetails.js]: Remove blank line

**Review by jangeroo — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (orangesmartie92) [reply src/App.js]: OK.

File: src/ItemDetails.js (hunk 3 above)
Comment (orangesmartie92) [reply src/ItemDetails.js]: ok

File: src/ItemDetails.js (hunk 2 above)
Comment (orangesmartie92) [reply src/ItemDetails.js]: ok

**Review by jangeroo — APPROVED** [review]

**Pull request merged**